Documentation/Doxygen/src/history.md

```diff
@@ -10,24 +10,19 @@ The table below provides information about the changes delivered with specific v
       <th>Description</th>
     </tr>
     <tr>
-      <td>V5.7.0</td>
```

Comment:
v5.7.0 was never released

Reply:
Therefore it is removed with this commit.